command-parser/src/parser.rs

```diff
@@ -152,6 +152,19 @@ mod tests {
         Parser::new(config)
     }
 
+    #[test]
+    fn double_command() {
+        let parser = simple_config();
+        match parser.parse("!echoecho") {
+            Some(Command {
+                name, ..
+            }) => {
+                assert_eq!("echoecho", name);
+            },
+            None => (),
+        }
+    }
```

Comment:
This should probably be rewritten as it will never hit the `assert_eq` on line 162, but instead will hit the None on line 164

Reply:
What would you think would be a better test to make sure stuff like repeated commands aren't caught as valid? (could also just remove the test, only added it to make sure the concern brought up in #74 was covered)

Reply:
Something like this could work
```rust
    #[test]
    fn double_command() {
        let parser = simple_config();
        match parser.parse("!echoecho") {
            Some(_) => panic!("Double match!"),
            None => (),
        }
    }
```

Reply:
👍